tests/testsuite.src/run_extensions.at

```diff
@@ -5219,6 +5219,22 @@ Error-Message:   prog.cob:20: module 'Tilt' not found
 AT_CLEANUP
 
 
+AT_SETUP([System routine x'91' function NN])
+AT_KEYWORDS([extensions])
+
+# to be done after 3.2rc1
+AT_SKIP_IF([true])
+
+AT_DATA([prog.cob], [
+])
+AT_CHECK([$COMPILE -std=mf prog.cob], [0], [], [])
+AT_CHECK([$COBCRUN_DIRECT ./prog], [1],
+[
+], [])
+
+AT_CLEANUP
```

Comment:
That's an interesting one...

Reply:
I guess this is a placeholder. Should I remove it ?
I've just moved my tests for function 46/47 and 48/49 just below it.